descwl_shear_sims/tests/test_ps_psf.py

```diff
@@ -0,0 +1,124 @@
+"""
+Copied from https://github.com/beckermr/metadetect-sims under BSD
+"""
+
+import numpy as np
+import galsim
+
+import pytest
+
+from ..ps_psf import PowerSpectrumPSF
+
+PIXEL_SCALE = 0.25
+
+
+def _get_fwhm_g1g2(psf_im):
+    mom = galsim.hsm.FindAdaptiveMom(psf_im)
```

Comment:
FYI, adaptive moments are prone to failure, if you change a seed it may well fail

Reply:
For sure.